CONTRIBUTING.md

```diff
@@ -1,32 +1,90 @@
+* `$KOS` is the full path to your KOS git repository.
+* Other than examples of full paths for Windows, all paths are shown in this
+  guide using the unix directory seperator of `/`.  Please change all instances
+  of `/` and `\` to match the style of your development environment.
+* You already have a C# IDE and are familiar with how to use it to build a
+  Visual Studio solution.
+
+####Setting Up Your repository
+1. Use the github web interface to create a fork of KSP-KOS/KOS
+2. Your fork should have a web address like `https://github.com/[username]/KOS`
+  and a git address like `https://github.com/[username]/KOS.git`
+3. If you use Github's desktop client, you can click the "Save to desktop"
+  button, otherwise using the git command line:
+  ```
+  git clone https://github.com/[username]/KOS.git
+  ```
+4. Set up KSP-KOS/KOS as the remote `upstream`:
+  ```
+  git remote add upstream https://github.com/KSP-KOS/KOS.git
+  ```
+5. Create a new branch based on `develop` whenever you make edits.
+  ```
+  git branch -m develop [new-branch-name]
+  ```
+6. Reserve your local `develop` branch for tracking the upstream (KSP-KOS/KOS)
+  `develop` branch.  You should update the local `develop` branch whenever there
+  is a change to the upstream branch:
+  ```
+  git pull --ff-only upstream develop
+  ```
+  If you have not made any local conflicting changes to the `develop` branch
+  itself, the "fast forward" pull will update your local branch without adding
+  a merge commit.
+7. If you need to update a local branch you are working on, you may "pull" the
+  upstream branch or rebase:
+  ```
+  git pull upstream develop
+  ```
+  or:
+  ```
+  git rebase upstream develop
+  ```
+  See the notes above regarding `rebase`
+8. You may push your branch with edits to your `origin` on github, and submit a
+  pull request to KSP-KOS/KOS `develop` for review to be included.
+
+####Setting Up The Solution Dependencies
 
-1. Copy `Resources/GameData/kOS` to `$KSP/GameData/`, `where $KSP` is your
-   Kerbal Space Program installation directory.
+1. Copy the folder `$KOS/Resources/GameData/kOS` to `$KSP/GameData/`
 
 2. Download the latest KSPAPIExtensions.dll from
-   https://github.com/Swamp-Ig/KSPAPIExtensions/releases, and copy 
-   it to `$KSP/GameData/kOS/Plugins`.
-
-3. Create a file at src/kOS/kOS.csproj.user conaining the following XML,
-   replacing /path/to/KSP with your own KSP install directory.
-
-        <?xml version="1.0" encoding="utf-8"?>
-        <Project ToolsVersion="4.0" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
-          <PropertyGroup>
-            <ReferencePath>
-              /path/to/KSP/KSP_Data/Managed;
-              /path/to/KSP/GameData/kOS/Plugins
-            </ReferencePath>
-          </PropertyGroup>
-        </Project>
-   
+   https://github.com/Swamp-Ig/KSPAPIExtensions/releases, and copy
+   it to `$KSP/GameData/kOS/Plugins` and `$KOS/Resources`.
```

Comment:
I will update the readme to put them into the `$KOS/Resources/GameData/kOS/Plugins` folder, and update the solution file's hint path to match (since the hint currently points to just `Resources`)